docs/CHANGELOG.md

```diff
@@ -0,0 +1,115 @@
+# Changelog
+
+All notable changes to the Animal-AI project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
I remember a conversation about this in one of the AnimalAI meetings but I can't remember the outcome. Do we definitely adhere to semantic versioning?

Reply:
Yes, a further revision of this doc will be implemented when i add more data as it is simply in it's infancy.